.github/renovate.json

```diff
@@ -247,7 +247,10 @@
         "windows/nanoserver",
         "windows/servercore",
         "containernetworking/azure-npm",
-        "windows/servercore/iis"
+        "windows/servercore/iis",
+        "oss/kubernetes/windows-gmsa-webhook",
+        "oss/kubernetes-csi/livenessprobe",
+        "oss/kubernetes/pause"
```

Comment:
This one we already manage for Linux. Are we sure it won't run into a conflict ?

Reply:
Done.